common/src/main/java/com/redhat/cloud/notifications/models/BehaviorGroup.java

```diff
@@ -93,8 +108,12 @@ public class BehaviorGroup extends CreationUpdateTimestamped {
 
     @JsonInclude
     @JsonProperty(access = READ_ONLY, value = "default_behavior")
-    public boolean isDefaultBehavior() {
-        return accountId == null;
+    public boolean isDefaultBehavior(boolean orgIdUsage) {
```

Comment:
Is this one working properly when returning behavior groups and default behavior groups?
IIUC this one is called automatically by the json encoder. Could we instead inject the configuration (either as a boolean field that is `@JsonIgnore'd`  or the OrgIDConfig object.

Reply:
either we don't have tests covering this or it's working properly.

Reply:
wtf.. :D what a "feature". when you add a parameter to this method, the "default_behavior" field in the JSON response get's ignored. It's just not there. And we did not have a test checking if default_behavior is part of the JSON response.